README-runner.md

```diff
@@ -0,0 +1,31 @@
+### swift-upload-runner – better browser file upload and download for swift-browser-ui
+
+`swift-upload-runner` makes it possible to properly batch upload from browser
+into object storage backends using Openstack Swift API. It will also make possible
+proper container uploads without plaintext passwords or tokens, to prevent
+token expiration.
+
+Additionally, the runner contains functionality for downloading whole containers
+as an archive without intermediary storage, as well as proxying downloads from
+shared containers, i.e. containers where the project has access via an ACL entry.
+
+### Installation
+`pip install git+https://github.com/cscfi/swift-upload-runner.git`
```

Comment:
this might get deprecated

Reply:
yup will be depregated, changed over to similar install to the other readmes